index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Hien's portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <img src="/image/logoipsum-265.svg" alt="Logo" class="logo">
+        <nav >
+            <div class="hamberger"></div>
+            <ul class="menu">
+                <li><a href="#portfolio">Porfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav> 
+    </header>
+    <main>
+        <section class="portfolio" id="portfolio">
+            <h1 class="main-header">I'm Hien <br> Glad to see you!</h1>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social">
+                <p>let's connect</p>
+                <div class="social-icon">
+                    <img src="./image/twister.png" alt="" class="contact-icon">
+                    <img src="./image/phone.png" alt="" class="contact-icon">
+                    <img src="./image/mail.png" alt="" class="contact-icon">
+                    <img src="./image/facebook.png" alt="" class="contact-icon">
+                </div>
```

Comment:
- Kindle, make contact lists inside `<ul>` instead of `<div>`  and make each `<img>` inside `<li>` as your previous code reviewer suggested you,